src/scripts/main.js

```diff
@@ -1 +1,12 @@
-'use strict';
+const largeImg = document.getElementById('largeImg');
+const thumbs = document.getElementById('thumbs');
+
+thumbs.addEventListener('click', (e) => {
+  e.preventDefault();
```

Comment:
The `e.preventDefault()` call is correct if you want to prevent the default action of the anchor tag. However, ensure that this behavior is desired for all clicks within the 'thumbs' element.